build fails with both `switchres` and `videocore` options

## Description

Trying to build RetroArch with `--enable-videocore` fails when `switchres` is also selected. Switchres is now enabled automatically I think (on Linux ?), so just enabling the VC option leads to a build failure.

### Expected behavior

Either the build to not fail or `configure` to fail first, warning the user that the two options are not compatible.

### Actual behavior

Build log:
```
CXX deps/switchres/monitor.cpp
gfx/video_crt_switch.c: In function ‘crt_switch_set_aspect’:
gfx/video_crt_switch.c:118:4: error: unknown type name ‘sr_state’; did you mean ‘si_stime’?
    sr_state state;
    ^~~~~~~~
    si_stime
CXX deps/switchres/modeline.cpp
gfx/video_crt_switch.c:138:4: warning: implicit declaration of function ‘sr_get_state’; did you mean ‘setstate’? [-Wimplicit-function-declaration]
    sr_get_state(&state);
    ^~~~~~~~~~~~
    setstate
gfx/video_crt_switch.c:140:31: error: request for member ‘super_width’ in something not a structure or union
    if ((int)srm_width >= state.super_width && !srm_isstretched)
                               ^
gfx/video_crt_switch.c: In function ‘crt_destroy_modes’:
gfx/video_crt_switch.c:376:7: warning: implicit declaration of function ‘sr_deinit’; did you mean ‘vcos_deinit’? [-Wimplicit-function-declaration]
       sr_deinit();
       ^~~~~~~~~
       vcos_deinit
gfx/video_crt_switch.c: In function ‘crt_adjust_sr_ini’:
gfx/video_crt_switch.c:472:10: warning: implicit declaration of function ‘sr_load_ini’ [-Wimplicit-function-declaration]
          sr_load_ini((char *)"switchres.ini");
          ^~~~~~~~~~~
At top level:
gfx/video_crt_switch.c:459:6: warning: ‘crt_adjust_sr_ini’ defined but not used [-Wunused-function]
 void crt_adjust_sr_ini(videocrt_switch_t *p_switch)
      ^~~~~~~~~~~~~~~~~
```

### Steps to reproduce the bug

1. Run configure with `./configure --enable-videocore` (needs the VC libraries in `/opt/vc`).
2. Build with `make`
3. Notice the build error.

### Bisect Results

Not attempted

### Version/Commit

RetroArch latest `master` , 64bcbbc5b0940.

### Environment information

- OS: Raspbian 10, `armhf`
- Compiler: `gcc` 8.3.0

--- a/gfx/video_crt_switch.c
+++ b/gfx/video_crt_switch.c
@@ -33,10 +33,8 @@
 #include "../paths.h"
 #include "gfx_display.h"
 
-#if !defined(HAVE_VIDEOCORE)
 #include "../deps/switchres/switchres_wrapper.h"
 static sr_mode srm;
-#endif
 
 #ifdef HAVE_CONFIG_H
 #include "../config.h"

